docs/kusama-endpoints.md

```diff
@@ -0,0 +1,84 @@
+
+## Parity Archive Node
+
+The development company Parity maintains an archive node at endpoint `wss://kusama-rpc.polkadot.io/`. 
+
+To connect to the Parity node, use the endpoint in your JavaScript apps like so:
+
+```javascript
+const{ ApiPromise, WsProvider } = require('@polkadot/api')
+
+(async () => {
+    const provider = new WsProvider('wss://kusama-rpc.polkadot.io/')
+    const api = await ApiPromise.create({ provider })
+    // ...
+```
+
+or in Polkadot Apps UI by clicking the top-left corner of the screen and selecting the appropriate option:
+
+![Selecting Parity's Node](/img/endpoints/parity.png)
+
+## Web3 Foundation Archive Node
+
+The Web3 Foundation maintains an archive node at endpoint `wss://cc3-5.kusama.network/`. 
+
+To connect to this node, use the endpoint in your JavaScript apps like so:
+
+```javascript
+const{ ApiPromise, WsProvider } = require('@polkadot/api')
+
+(async () => {
+    const provider = new WsProvider('wss://cc3-5.kusama.network/')
+    const api = await ApiPromise.create({ provider })
+    // ...
+```
+
+or in Polkadot Apps UI by clicking the top-left corner of the screen and selecting the appropriate option:
+
+![Selecting W3F's Node](/img/endpoints/w3f.png)
+
+## Kusama RYO Full Node Cloud
+
+RYO (Run Your Own) infrastructure from [Avado](https://ava.do) is a load balancer - a server redirecting requests randomly to other computers - to which you can attach your own home-run node. This means that by connecting to the load balancer's endpoint, you're randomly connecting to another person's public node: the convenience of one reliable endpoint, without the downside of centralization\*.
+
+Check the current status of the Kusama RYO cloud including available nodes at https://status.cloud.ava.do/
+
+To connect to the RYO cloud, use the endpoint `wss://kusama.polkadot.cloud.ava.do/` in your JavaScript apps like so:
+
+```javascript
+const{ ApiPromise, WsProvider } = require('@polkadot/api')
+
+(async () => {
+    const provider = new WsProvider('wss://kusama.polkadot.cloud.ava.do/')
+    const api = await ApiPromise.create({ provider })
+    // ...
+```
+
+or in Polkadot Apps UI by clicking the top-left corner of the screen and selecting the appropriate option:
+
+![Selecting the RYO cloud](/img/endpoints/ryo.png)
+
+### Things to know
+
+#### Reliability 
+
+There is no incentive to run these public nodes yet. We're working on making that happen. Ideally, the Kusama Treasury would fund this public infrastructure venture. For now, running the nodes is altruistic which means the nodes can disappear for no reason.
```

Comment:
I think the last sentence isn't really true - incentivized nodes can also disappear for no reason.  

```suggestion
There is no incentive to run these public nodes yet. We're working on making that happen. Ideally, the Kusama Treasury would fund this public infrastructure venture. For now, running the nodes is altruistic.
```